driver_scripts/setup_MOC_basins.py

```diff
@@ -19,6 +19,8 @@ def spcall(args): #{{{
 
 options, args = parser.parse_args()
 
+groupNameMOC = 'MOCRegionsGroup'
+groupNameBasins = 'BasinRegionsGroup'
```

Comment:
I'm wondering if there isn't a choice of better name for `groupNameBasins`.  We probably want to make sure the group names are unique throughout the repo too.